cmd/porter/main.go

```diff
@@ -1,3 +1,5 @@
+//go:generate packr2
```

Comment:
This is what was missing and causing things to not work properly for binaries not build on dev machines. Thanks for figuring that out! 👍 

It can be hard to tell because packr will fallback to the filesystem when the data isn't found in the binary. Because this line was missing, nothing was being built into the binary for the help text. But it was working on dev machines because the files were present locally. 😆 

The other changes in the file aren't necessary and should be reverted.

Reply:
The other changes are intentional.
I thought directory name: `templates`  makes sense than `helptext` because in future we may need to add other templates. So, i refactored that logic

If i am wrong/misunderstood , i can revert if you want

Reply:
Our preference for pull requests, is it [limit them to a single task](https://github.com/deislabs/porter/blob/master/CONTRIBUTING.md#how-to-get-your-pull-request-reviewed-fast). In this case the issue is to fix the help text and since the refactoring wasn't necessary for the fix, we prefer to not include it in the same PR.

Later, if we had an issue where we needed to add more templates to cmd/porter, that would be the time to refactor. Right now we don't have anything like that on the backlog though.